### PR TITLE
Feature/create event service and dto

### PR DESCRIPTION
**Pull request by malyshevhen**

### CODE REVIEW

File: src/main/java/baza/trainee/service/impl/EventServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    @Override
+    public Event save(EventPublication newEvent) {
+        Event event = eventMapper.toEvent(newEvent);
+
+        return eventRepository.save(event);
+    }
+
+    @Override
+    public Event update(String id, EventPublication updatedEvent) {
+        Event event = eventMapper.toEvent(updatedEvent);
```
Comment (Anatolii-Omelchenko) [inline src/main/java/baza/trainee/service/impl/EventServiceImpl.java]: check if event is not present

File: src/main/java/baza/trainee/service/impl/EventServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        event.setId(id);
+
+        return eventRepository.update(event);
+    }
+
+    @Override
+    public void deleteEventById(String id) {
```
Comment (Anatolii-Omelchenko) [inline src/main/java/baza/trainee/service/impl/EventServiceImpl.java:53]: check if event is not present

File: src/main/java/baza/trainee/service/impl/EventServiceImpl.java (hunk 1 above)
Comment (malyshevhen) [reply src/main/java/baza/trainee/service/impl/EventServiceImpl.java]: Done

File: src/main/java/baza/trainee/service/impl/EventServiceImpl.java (hunk 2 above)
Comment (malyshevhen) [reply src/main/java/baza/trainee/service/impl/EventServiceImpl.java:53]: Done

**Pull request merged**